packages/website/package.json

```diff
@@ -20,9 +21,9 @@
     "@tailwindcss/jit": "0.1.18",
     "autoprefixer": "^10.2.5",
     "deepmerge": "^4.2.2",
+    "patch-package": "^6.4.7",
```

Comment:
No need.

Already in the root deps & this is not a website project deps.

All the devdeps are moved to root deps, to manage single dev deps versioning.